src/ResourceManagement/Sql/Sql.Tests/ScenarioTests/Sql2.AdvisorScenarioTest.cs

```diff
@@ -0,0 +1,369 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Azure.Management.Resources.Models;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using Microsoft.Azure.Test;
+using Xunit;
+
+namespace Sql2.Tests.ScenarioTests
+{
+    /// <summary>
+    /// Contains tests for Server, Elastic Pool and Database Advisors
+    /// </summary>
+    public class Sql2AdvisorScenarioTests : TestBase
+    {
+        private const string ResourceGroupName = "WIRunnersProd";
```

Comment:
You have json files, which indicate that you ran these tests against live azure. It looks like youi set up a bunch of Sql asssets in some subscription and then just ran list cmdlets against this setup.  If this is the case, the next person who has to make a change here will have a great deal of difficulty in running these tests live for new recordings - they will basically have to either replicate exactly the setup you had live, or they will have to change the code.

If these changes are time sensitive, it is OK to check-in and add the test setup as part of the tests within the next month.  If you want to do this, please file a bug for this in github, assign to yourself and reference it here.